src/admin-views/aggregator/tabs/import-form.php

```diff
@@ -1,5 +1,6 @@
 <?php
-$has_license_key = Tribe__Events__Aggregator::is_service_active();
+$has_license_key = tribe( 'events-aggregator.main' )->is_service_active();
```

Comment:
We have a bunch of other places that we use this structure, I would rather make `is_service_active` a non-static method.